bin/build-docs-site.sh

```diff
@@ -11,6 +11,11 @@
 # Set `-e` so that non-zero exit code from any step will be honored
 set -e
 
+if ! [ -x "$(command -v bundle)" ]; then
```

Comment:
Is `command` available on Linux? What about `which bundle`?

Reply:
Answer in stackoverflow: https://stackoverflow.com/a/677212/11729993
Command in my linux (Solus OS): https://gofile.io/?c=zycrvN

Reply:
I think it's a small modification that will be useful. For example, I didn't know what it was and I had to investigate online